File: doc/.blinky/blinky.pas
PROGRAM blinky(input, output);

USES
  SysUtils,
  GPIO,
  GPIO_libsimpleio,
  RaspberryPi;

VAR
  LED : GPIO.Pin;

BEGIN
  Writeln;
  Writeln('MuntsOS Free Pascal LED Test');
  Writeln;

  { Configure a GPIO output to drive an LED }

  LED := GPIO_libsimpleio.PinSubclass.Create(GPIO26, Output, False);

  { Flash the LED forever (until killed) }

  Writeln('Press CONTROL-C to exit.');
  Writeln;

  REPEAT
    LED.state := NOT LED.state;
    sleep(500);
  UNTIL False;
END.

File: doc/.blinky/blinky.py
#! /usr/bin/python3

from munts.libsimpleio.gpio        import Pin, Direction
from munts.libsimpleio.raspberrypi import GPIO26

import time

print("\nMuntsOS Python3 LED Test\n")

LED = Pin(GPIO26, Direction.Output)

while True:
  LED.state = not LED.state
  time.sleep(0.5)
